lib/perl/Genome/Model/Build.pm

```diff
@@ -1395,6 +1395,11 @@ sub _launch {
     local $ENV{UR_DUMP_STATUS_MESSAGES} = 1;
     local $ENV{UR_COMMAND_DUMP_STATUS_MESSAGES} = 1;
 
+    # we don't use 'local' because we want this to persist through
+    # spawned workflows
+    $ENV{Genome::Config::spec('lsf_project_name')->env} =
+        sprintf("build/%s", $self->id);
```

Comment:
> I would guess the problem with localization is because the workflow is finally submitted in a commit observer. The project variable needs to be set at that time to work properly.

That makes perfect sense.  Thanks for pointing that out.